pages/about.html

```diff
@@ -0,0 +1,142 @@
+                <fieldset>
+                    <legend>Send me a quick message</legend>
+                    Your Name:
+                    <input type="text" name="fullname" maxlength="30" placeholder="Fullname" required>
+                <label>
+                    Your email:
+                    <input id="mail" class="email" type="email" name="email" placeholder="Email address">
+                  </label>
+                    <label>
+                        Your message:  
+                        <textarea name="comment" required placeholder="Type your comment here" cols="30" rows="10"></textarea>
+                    </label>
+                    <span class="message"></span>
+                    <!-- your other form fields go here -->
+                    <button type="submit">Send</button>
+                </fieldset>
+            </form>
+        </div>
+    </div>
+        <div class="about-2">
+            <div class="about-desc-1">
+                <h3>Creative Commons Global Summit 2015 Logo</h3>
+                <div><hr class="about-hr"></div>
+                <p>The logo of CC Summit 2015 was decided thrpugh
+                     the logo competition from 8 June to 7 July.
+                    <span id="span-1">After online public voting and selection
+                     Committe's judgng process, the logo from Naresh
+                      Agrawal won the prize. After consultaion with 
+                      Mr. Agrawal we optimized the prie-winnin-logo 
+                      for this year's summit as following:</span></p>
+            </div>
+             <div class="about-logo">
+                <img src="../images/logo.png" alt="logo">
+             </div>
+             <hr>
+             <div class="about-desc-2">
+                <h3>See the past CC Global Summits</h3>
+                <div><hr class="about-hr"></div>
+                <p>Take a look at the two CC Gloal 
+                Summits which took place in Buenos Airies and Warsaw.</p>
+             </div>
+    
+                <div class="past-events">
+                    <img src="../images/w3.jpeg" alt="Past Events">
+                    <img src="../images/w2.jpeg" alt="Past Events">
+                </div>
+                <div class="footer-1">
+                <div>
+                    <div class="partners">
+                      <img src="../images/mozilla.jpeg" alt="Partner">
+                      <img src="../images/google.jpeg" alt="Partner">
+                        <img src="../images/naver.jpeg" alt="Partner">
+                        <img src="../images/airbnb.jpeg" alt="Partner">
+                    </div>
+                </div>
+                </div>
+                    <div id="footer-logo" class="footer-2">
+                        <img src="../images/logo.png" alt="logo">
+                        <p>2015 Creative Commons Korea.
+                        Some Rights Reserved.</p>
+                    </div>
```

Comment:
Kindly replace this `div` tag with `footer` tag since this block is the footer of your page 🤗